pkg/kv/kvclient/kvtenant/connector.go

```diff
@@ -880,6 +880,22 @@ func (c *connector) HotRangesV2(
 	return resp, nil
 }
 
+// DownloadSpan implements the serverpb.TenantStatusServer interface
+func (c *connector) DownloadSpan(
+	ctx context.Context, req *serverpb.DownloadSpanRequest,
+) (*serverpb.DownloadSpanResponse, error) {
+	var resp *serverpb.DownloadSpanResponse
+	r := *req
```

Comment:
I'm guessing this came from the HotRangesV2 code above. But I _think_ that we copy this in that request only because we want to modify it as part of the request.